src/components/screens/EventRecords.js

```diff
@@ -375,6 +393,20 @@ export const EventRecords = ({ navigation, route }) => {
         }
         showBackToTop
       />
+      {eventListIntro?.buttonType == EVENT_SUGGESTION_BUTTON.BOTTOM_FLOATING && (
+        <EventSuggestionButton
+          buttonTitle={eventListIntro.buttonTitle}
+          onPress={() => {
+            if (eventListIntro.url) {
+              openLink(eventListIntro.url, openWebScreen);
+            } else {
+              navigation.navigate(ScreenName.EventSuggestion, {
+                formIntroText: eventListIntro.formIntroText
+              });
+            }
+          }}
+        />
+      )}
```

Comment:
i cannot overscroll the button to see the last entry in the list.
compared to pois with floating button, where i can overscroll to see the bottom last entry of the list entirely

|events|pois|
|---|---|
|![Simulator Screen Recording - iPhone 16 - 2024-12-03 at 15 59 38](https://github.com/user-attachments/assets/d7d2abb0-4ac3-49b8-b4fc-e7ddb56d62d8)|![Simulator Screen Recording - iPhone 16 - 2024-12-03 at 16 02 47](https://github.com/user-attachments/assets/5d75cf32-29af-4652-8d7e-a6bcf37e3cbc)|

please add the "padding" after the list as well for events, if floating button is set